realis.go

```diff
@@ -276,14 +284,17 @@ func NewRealisClient(options ...ClientOption) (Realis, error) {
 
 	// Turn off all logging (including debug)
 	if config.logger == nil {
-		config.logger = &LevelLogger{NoopLogger{}, false}
+		config.logger = &LevelLogger{NoopLogger{}, false, false}
 	}
 
 	// Set a logger if debug has been set to true but no logger has been set
 	if config.logger == nil && config.debug {
-		config.logger = &LevelLogger{log.New(os.Stdout, "realis: ", log.Ltime|log.Ldate|log.LUTC), true}
+		config.logger = &LevelLogger{log.New(os.Stdout, "realis: ", log.Ltime|log.Ldate|log.LUTC), true, false}
```

Comment:
I think it would be better to use the full form for initialize struct here, omit the fields name may have unexpected behavior and it's hard to tell what those flags are for

Reply:
Agreed, weighing the options, this made the most sense for V1. We'll come up with something more clever for V2 😄